sdk-workflows/src/main/java/io/dapr/workflows/runtime/Workflow.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2023 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.workflows.runtime;
+
+import com.google.protobuf.Empty;
+import reactor.core.publisher.Mono;
+
+/**
+ * Common interface for workflow implementations.
+ */
+public abstract class Workflow {
```

Comment:
This Workflow class and related WorkflowContext/DaprWorkflowContextImpl should move the its parent package "io.dapr.workflows". Because they are basic definition for workflow, not for workflow runtime.